out/can_handlers.h

```diff
@@ -6,7 +6,7 @@
 #include <stdint.h>
 #include <stddef.h>
 
-#define CAN_HANDLERS_TABLE_SIZE 6
+#define CAN_HANDLERS_TABLE_SIZE 8
```

Comment:
Following previous comment, this will need to be split into a size for the CAN-C handlers table and a size for CAN-S handlers table.